Fix for Issue 158

ajogseneca opened the pull request:
## Additions
- Added test cases with the help of `python` and `selenium`.
- Make sure to have `python` installed and  `selenium` . It can be installed with `pip install selenium`

## Test checks
Test functions include :
- `def test_title_input_updates_preview(driver)` : It checks if the user inputed title is getting added to the form and also visible in the preview button.
- `def test_content_input_updates_preview(driver)` : It checks if the user inputed content of the blog is read by the form and also visible to the user when the preview button is pressed.
- `def test_image_upload_and_preview(driver)` : It checks if the file upload for the blog cover works, that is if the image is getting loaded in the form or not.
- `def test_preview_toggle(driver)` : This case checks if the preview button is working or not.

gaurovgiri reviewed (commented):
Oh wait, you last three PRs are related to same feature but unfortunately, these seemed a bit out of our plans for now.

gaurovgiri commented:
Let me know others thought on this and then we can proceed to approval

ajogseneca commented:
## Changes
- Added changes from the previous PR's review
- Changed the look of blog page style and scaled to 80%
- Switched to Google Forms to collect review questions.
- Changed test case to work with Google Form

New Blog Page style :
![image](https://github.com/user-attachments/assets/48ebc138-c696-4af5-89ab-5174b960d30c)

## Concern 
Google Forms' won't allow embedded link if there is a file upload present.
